### ISSUE TITLE
[add-resource] New Plugin

### ISSUE THREAD

**Issue by cron-ix** [getgrav/grav#3346]
I would like to add my new plugin to the Grav Repository.
Here are the project details: **[cron-ix/grav-plugin-pi-cpu-temp](https://github.com/cron-ix/grav-plugin-pi-cpu-temp)**

If GRAV is running on a Raspberry Pi, the plugin can show the cpu core temp in degree celsius

Thanx,
Thomas

**Comment by pamtbaau** [comment]
@cron-ix, I have had a quick view at your plugin and noticed the following:
- The plugin does not conform the plugin guides set by Grav
  - It does not have a `composer.json` file
  - It uses `require` instead of composer autoloader

  See [Plugins tutorial](https://learn.getgrav.org/17/plugins/plugin-tutorial) and the Developer section in the [1.7 Upgrade guide](https://learn.getgrav.org/17/advanced/grav-development/grav-17-upgrade-guide)
  The easiest way to conform to these guides is by creating a plugin using the Devtools plugin: 
   ```
   $ bin/gpm install devtools
   $ bin/plugin devtools new-plugin
   ```

- The Twig extension is loaded at every request including Admin and CLI. 
See the `onPluginsInitialized()` event-handler in the plugin generated by Devtools and above mentioned guides.
- The temperature show on the page will not be updated when page has been cached.
- Instead of returning an Exception message thrown by the system, a more descriptive message might be more helpful to the user.
- README:
  - Instructions should mention how to circumvent the caching issue mentioned above.
  - "Set Process Twig to true for your page" could be more descriptive using a snippet.

**Comment by cron-ix** [comment]
@pamtbaau Thank you for your review.

I used this [plugin recipe](https://learn.getgrav.org/17/cookbook/plugin-recipes) to build my plugin. Now I updated the code with devtools newplugin.

* composer.json added
* README.md updated with detailed information on how to to configure and use (eg disable caching)
* used 'file_get_contents()' with detailled error handling/reporting (thx for the hint - it can be that easy...), if file not found or return value is NULL

repo updated and new initial release

Thomas
